docs/getting-started.md

```diff
@@ -49,7 +51,9 @@ If you are facing a issue, please read the [FAQ](faq.html) first.
 {: .important } 
 > When asking for help, ask in English. We can't help you if we don't understand the question. Using a translator is fine.
 
-**Do not use manufacturer's model name or serial number when asking for help** (i.e: HP Chromebook 14a), it doesn't help with identifying the machine. Provide the boardname, otherwise your support request will be ignored. If you don't know where to find your boardname, take a look [here.](system-info.html)
+**Do not use manufacturer's model name or serial number when asking for help** (i.e: HP Chromebook 14a), it doesn't help with identifying the machine. Provide the boardname, otherwise your support request will be ignored. 
+
+If you don't know where to find your boardname, take a look [here.](system-info.html)
```

Comment:
(doesnt really matter)

Should the period be part of the link?

Reply:
We should never include the . in the link